Client/src/Pages/Uptime/Details/index.jsx

```diff
@@ -100,7 +100,7 @@ const DetailsPage = ({ isAdmin }) => {
 			}
 		};
 		fetchCertificate();
-	}, [authToken, monitorId, monitor]);
+	}, [authToken, monitorId, monitor, dateRange, uiTimezone, dateFormat]);
```

Comment:
Why did we add these 3 dependencies?

Reply:
I believe they are used in the hook, so we probably want the hook to run again if those values change?

Reply:
Oh yeah, I found two of them but not dateRange

Reply:
> Oh yeah, I found two of them but not dateRange

You're right, good catch!  I must have fat-fingered it in by accident.  Removed and commited, thank you!